feature/system/management/README.md

```diff
@@ -0,0 +1,146 @@
+# MGT-1: Management HA solution test
+
+## Summary
+
+- Test management HA
+
+## Testbed type
+
+*   https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_4.testbed
+
+## Procedure
+
+### Applying configuration
+
+For each section of configuration below, prepare a gnmi.SetBatch with all the configuration items appended to one SetBatch. Then apply the configuration to the DUT in one gnmi.Set using the `replace` option
+
+### Initial Setup:
+
+*   Connect DUT port-1, 2, 3 and 4 to ATE port-1, 2, 3 and 4
+*   Create VRF "mgmt" on DUT
+    *   /network-instances/network-instance[name=mgmt]/vrfs/vrf[name=mgmt]/config/name = mgmt
+    *   /network-instances/network-instance[name=mgmt]/vrfs/vrf[name=mgmt]/config/route-distinguisher = 64512:100
```

Comment:
Doesn't seems like there is such OC schema https://openconfig.net/projects/models/schemadocs/jstree/openconfig-network-instance.html

cc: @rszarecki

Reply:
fixed, I made an error copying/pasting paths